functions/tests/integration/test_location_with_firestore.py

```
@@ -0,0 +1,346 @@
+
+# =============================================================================
+# Integration Tests
+# =============================================================================
+
+
+@pytest.mark.skipif(not EMULATOR_HOST, reason=SKIP_REASON)
+class TestLocationWithFirestore:
+    """Integration tests using Firestore emulator."""
+
+    @pytest.mark.asyncio
+    async def test_end_to_end_location_lookup(self, seed_test_data):
```

Ruff (0.14.8)

Unused method argument: `seed_test_data`

(ARG002)